app/src/main/res/layout/activity_article_details.xml

```diff
@@ -1,87 +1,118 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="fill_parent"
     android:layout_height="fill_parent">
     <!-- Here you put the rest of your current view-->
 <androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:padding="8dp">
 
-    <TextView
-        android:layout_width="match_parent"
+    <WebView
```

Comment:
This is all what is needed and maybe add a title in the tool bath would reflect the news title. Add a text view within the toolbar and display the article title there. As for the content leave it to webview.